app/models/content_item.rb

```diff
@@ -2,8 +2,10 @@ class ContentItem
   attr_reader :content_item_data
 
   def self.find!(base_path)
-    response = Services.content_store.content_item(base_path)
-    new(response.to_hash)
+    content_item = Services.content_store.content_item(base_path)
+    content_item_hash = content_item.to_h
+    content_item_hash["cache_control"] = { "max_age" => content_item.cache_control["max-age"], "public" => content_item.cache_control.include?("public") }
```

Comment:
😄 cheeky, mutating the content item - it's unorthodox (you could keep the provided `content_item` class around and generate the hash as needed later) but it'll work.

here you could do `"public" => !content_item.cache_control.private?`

Reply:
Is that a better way of doing things? it will mean more code changing if I keep the object, I'll see if I can get that working